lib/sandthorn/aggregate_root_base.rb

```diff
@@ -118,6 +118,7 @@ def aggregate_build events
             current_aggregate_version = events.last[:aggregate_version]
             aggregate.send :set_orginating_aggregate_version!, current_aggregate_version
             aggregate.send :set_current_aggregate_version!, current_aggregate_version
+            aggregate.send :set_aggregate_id, events.first[:aggregate_id]
```

Comment:
Can a valid event lack `:aggregate_id`? If not: let's use `fetch` and throw an error when it's missing.